### PR TITLE
adressing Remove semistandard #712

### PR DESCRIPTION
**Pull request by smeet07**
### Background 
I don't think semistandard is being used in a meaningful way therefore removed it as a devdependency

### Fixes 
Remove semistandard #712

### CODE REVIEW

File: src/currencyservice/package.json [hunk 1]
```diff
@@ -20,7 +20,4 @@
     "request": "2.88.2",
     "xml2js": "0.4.23"
   },
-  "devDependencies": {
```
Comment (mathieu-benoit) [inline src/currencyservice/package.json:23]: I also see `"lint": "semistandard *.js"` line 8, does this need to be removed too?

Comment (bourgeoisor) [reply src/currencyservice/package.json:23]: I'd assume so if we're removing the module entirely

**Comment by smeet07** [comment]
yes I think we could remove that as well

**Comment by smeet07** [comment]
since we are removing semistandard module, the script wont work anyways

**Review by bourgeoisor — APPROVED** [review]
LGTM

**Comment by mathieu-benoit** [comment]
Thanks for your contribution @smeet07, much appreciated!

**Comment by smeet07** [comment]
No problem, Always happy to help

**Pull request merged**